guides/workload-autoscaling/helmfile.yaml.gotmpl

```diff
@@ -0,0 +1,191 @@
+      - ms-inference-scheduling/values_cpu.yaml
+    {{- else if eq .Environment.Name "digitalocean" }}
+      - ms-inference-scheduling/digitalocean-values.yaml
+    {{- else if eq .Environment.Name "xpu" }}
+      - ms-inference-scheduling/values_xpu.yaml
+    {{- else }}
+      - ms-inference-scheduling/values.yaml
+    {{- end }}
+    {{- if (eq .Environment.Name "istioBench") }}
+      - routing: 
+          {{ .Environment.Values.routing | toYaml | nindent 10 }}
+    {{- end }}
+    set:
+    # apply release name derived values
+      - name: "routing.inferencePool.name"
+        value: {{ printf "gaie-%s" $rn | quote }}
+      - name: "routing.parentRefs[0].name"
+        value: {{ printf "infra-%s-inference-gateway" $rn | quote }}
+    {{- if or (eq .Environment.Name "gke") (eq .Environment.Name "gke_tpu") }}  
+      - name: "decode.monitoring.podmonitor.enabled"
+        value: false
+    {{- end }}
+    labels:
+      kind: inference-stack
+{{- if eq .Environment.Name "wva" }}
+  # Prometheus Adapter - Required for WVA to expose custom metrics
+  - name: prometheus-adapter
+    namespace: {{ env "MON_NS" | default "monitoring" }}
+    chart: prometheus-community/prometheus-adapter
+    version: 4.0.1
+    installed: true
+    needs:
+      - {{ printf "ms-%s" $rn | quote }}
+    labels:
+      kind: autoscaling
+    {{- if env "PROMETHEUS_ADAPTER_VALUES" }}
+    values:
+      - {{ env "PROMETHEUS_ADAPTER_VALUES" }}
+    {{- end }}
+  # Workload Variant Autoscaler - Dynamic autoscaling for llm-d
+  # Note: WVA chart can be installed from a local path or OCI registry
+  # Set WVA_CHART_PATH to use a local chart (e.g., after cloning the repo)
+  # Otherwise, it will attempt to use the OCI registry path
+  - name: workload-variant-autoscaler
+    namespace: workload-variant-autoscaler-system
+    {{- if env "WVA_CHART_PATH" }}
+    chart: {{ env "WVA_CHART_PATH" }}
+    {{- else }}
+    chart: oci://ghcr.io/llm-d-incubation/workload-variant-autoscaler/charts/workload-variant-autoscaler
+    {{- end }}
+    version: {{ env "WVA_RELEASE" | default "v0.0.5" }}
+    installed: true
+    needs:
+      - {{ printf "ms-%s" $rn | quote }}
+      - prometheus-adapter
+    labels:
+      kind: autoscaling
+    set:
+      - name: va.accelerator
+        value: {{ env "WVA_ACCELERATOR" | default "L40S" | quote }}
+      - name: llmd.modelID
+        value: {{ env "WVA_MODEL_ID" | default "Qwen/Qwen3-0.6B" | quote }}
+      - name: vllmService.enabled
+        value: true
+      {{- if env "WVA_NODE_PORT" }}
+      - name: vllmService.nodePort
+        value: {{ env "WVA_NODE_PORT" }}
+      {{- end }}
+      - name: llmd.namespace
+        value: {{ $ns | quote }}
+      {{- if env "PROMETHEUS_CA_CERT_PATH" }}
+      - name: wva.prometheus.caCert
+        value: {{ env "PROMETHEUS_CA_CERT_PATH" | quote }}
+      {{- end }}
+    {{- if env "WVA_VALUES_FILE" }}
+    values:
+      - {{ env "WVA_VALUES_FILE" }}
```

Comment:
We should pick default behaviours - none of this should rely on ENV variabls (except maybe like release postfix which would apply to all charts in concert).

Reply:
done